### PR TITLE
docs(http): add operation log endpoints

### LINKED ISSUE #11359: api: add swagger documentation about operation log endpoints
TODO:
- [ ] `/api/v2/users/:id/log`
- [ ] `/api/v2/buckets/:id/log`
- [ ] `/api/v2/orgs/:id/log`
- [ ] `/api/v2/dashboard/:id/log`

### PR DESCRIPTION
**Pull request by zhulongcheng**
Closes #11359

_Briefly describe your proposed changes:_

  - [x] Rebased/mergeable
  - [x] Tests pass
  - [x] http/swagger.yml updated (if modified Go structs or API)
  - [x] Sign [CLA](https://influxdata.com/community/cla/) (if not already signed)

### CODE REVIEW

**Comment by goller** [comment]
Hey @zhulongcheng , I hope all is well !

Would you rebase onto master?  I believe the gotest failures will be fixed once you do so.

File: http/swagger.yml [hunk 1]
```diff
@@ -4984,6 +5100,30 @@ components:
...
+        links:
+          type: object
+          properties:
+            user:
+              $ref: "#/components/schemas/Link"
+    OperationLogs:
+      type: object
+      properties:
+        logs:
+          $ref: "#/components/schemas/OperationLog"
```
Comment (goller) [inline http/swagger.yml]: @desa should this be an array of operation logs?

Comment (desa) [reply http/swagger.yml]: Yup. Array seems right to me.

Comment (goller) [reply http/swagger.yml]: @zhulongcheng would you change this to an array rather than an object?

**Pull request merged**